### ISSUE TITLE
Can't get it to work

### ISSUE THREAD

**Issue by zenopopovici** [se-panfilov/bower-check-updates#8]
Will not work at all. This is what I'm getting. I've tried both the alpha and stable versions.

```
Unhandled rejection Error: 404 Not Found
    at Object.ensureErrorObject (/usr/local/lib/node_modules/bower-check-updates/node_modules/bluebird/js/main/util.js:261:20)
    at Promise._rejectCallback (/usr/local/lib/node_modules/bower-check-updates/node_modules/bluebird/js/main/promise.js:465:22)
    at PromiseArray.__hardReject__.PromiseArray._reject (/usr/local/lib/node_modules/bower-check-updates/node_modules/bluebird/js/main/promise_array.js:109:19)
    at Promise._settlePromiseAt (/usr/local/lib/node_modules/bower-check-updates/node_modules/bluebird/js/main/promise.js:575:21)
    at Promise._settlePromises (/usr/local/lib/node_modules/bower-check-updates/node_modules/bluebird/js/main/promise.js:693:14)
    at Async._drainQueue (/usr/local/lib/node_modules/bower-check-updates/node_modules/bluebird/js/main/async.js:123:16)
    at Async._drainQueues (/usr/local/lib/node_modules/bower-check-updates/node_modules/bluebird/js/main/async.js:133:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/usr/local/lib/node_modules/bower-check-updates/node_modules/bluebird/js/main/async.js:15:14)
    at processImmediate [as _immediateCallback] (timers.js:367:17)
```

**Comment by se-panfilov** [comment]
@zenopopovici Thanks for reporting.
Fixed this issue in `2.1.0`. Please check the update

**Comment by zenopopovici** [comment]
Works now. Thanks!
